airbyte-webapp/src/views/Connection/ConnectionForm/components/SyncCatalogField.module.scss

```diff
@@ -7,4 +9,5 @@
   font-weight: 500;
   font-size: 14px;
   line-height: 17px;
+  padding: variables.$spacing-xl;
```

Comment:
This gives it a huge padding to the top and bottom, maybe it's meant to be just the horizontal padding?

![Screen Shot 2022-11-09 at 09 39 44](https://user-images.githubusercontent.com/168664/200861064-3ab76b63-0a97-4926-bba4-1c2a65a22274.png)